support/modules-artifact-gen/single-file-artifact-gen

```diff
@@ -155,8 +155,16 @@ echo "$dest_dir" > $dest_dir_file
 # Create single_file file in plain text
 echo "$filename" > $filename_file
 
+set +e;
+STAT_HAS_f=1;
+stat -f %A "${file}" >/dev/null 2>&1 || STAT_HAS_f=0;
+if [ $STAT_HAS_f -eq 1 ]; then
+ stat -f %A "${file}" | grep -qE '^[0-7]{3,4}$' || STAT_HAS_f=0;
```

Comment:
This second check is redundant and not caching anything extra IMO: There might be numeric results and still not be the permissions in octel format (for instance, using %a)

Furthermore, @mzedel also suggested not to add it in his comment.